buildspec-release.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      # prepare the release (update versions, changelog etc.)
+      - git-release --prepare
+
+      # run linters
+      - tox -e flake8,pylint
+
+      # run package and docbuild checks
+      - tox -e twine
+      - tox -e sphinx
+
+      # run unit tests
+      - AWS_ACCESS_KEY_ID= AWS_SECRET_ACCESS_KEY= AWS_SESSION_TOKEN=
+        AWS_CONTAINER_CREDENTIALS_RELATIVE_URI= AWS_DEFAULT_REGION=
+        tox -e py27,py36 -- tests/unit
+
+      # consider running integration and tests
```

Comment:
maybe the tests annotated with `@canary_quick`?

also, I think you mean "integration tests" not "integration and tests"?

Reply:
fixed